style.css

```diff
@@ -131,4 +132,291 @@ img {
+    font-family: var(--font-family-heading);
+    font-weight: 400;
+    letter-spacing: 0.02em;
+    text-align: center;
+}
+
+.grand {
+    color: var(--color-accent-red);
+}
+
+.players {
+    color: var(--color-accent-yellow);
+}
+
+.about-main-part{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 20px;
+    height: 180px;
+}
+
+.about-main-part-text{
+    max-width: 460px;
+    margin: 0;
+}
+
+.img-live{
+    width: 320px;
+    height: 180px;
+}
+
+@media(max-width:750px){
+    .about{
+        padding: 40px 20px;
+    }
+
+    .about-main-copy{
+        font-size: 24px;
+    }
+
+    .about-main-part{
+        gap: 0;
+    }
+
+    .about-main-part-text{
+        font-family: var(--font-family-about-mobile);
+        max-width: 750px;
+    }
+
+    .about-main-part-img-wrapper{
+        display: none;
+        width: 0px;
+    }
+}
+
+/*Courses*/
+.courses {
+    margin: 0;
+    background-color: var(--color-background-naby);
+    padding: 80px 20px;
+    color: var(--color-white);
+    font-family: var(--font-family-body);
+    background-size: cover;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+}
+
+.courses-container{
+    max-width: 800px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+}
+
+.courses-title{
+    margin: 0;
+    font-size: 32px;
+    font-family: var(--font-family-heading);
+    font-weight: 400;
+    letter-spacing: 0.02em;
+    text-align: center;
+}
+
+/*Cards*/
+.courses-cards-container{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    gap: 20px;
+}
+
+.courses-cards{
+    background-color: var(--color-accent-yellow);
+    box-shadow: 4px 4px 0px var(--color-accent-red);
+    height: 150px;
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    text-decoration: none;
+    transition-property: "background-color" "box-shadow";
+    transition-duration: 300ms;
+    transition-timing-function: linear;
+}
+
+.courses-cards:hover{
+    background-color: var(--color-course-hover-yellow);
+    box-shadow: 12px 12px 0px var(--color-button-hover-red);
+}
+
+.courses-cards-img-wrapper{
+    background-image: url("./img/live.png");
+    max-width: 250px;
+    min-width: 250px;
+    height: 100%;
+    background-size: cover;
+    position: relative;
+}
+
+.courses-cards-img-overlay{
+    width: 100%;
+    height: 100%;
+    background-color: var(--color-course-overlay);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.courses-cards-title{
+    line-height: 48px;
+    margin: 0;
+    font-size: 32px;
+    font-family: var(--font-family-heading);
+    font-weight: 400;
+    letter-spacing: 0.05em;
+    color: var(--color-white);
+}
+
+.courses-cards-text-container{
+    margin: 0;
+    padding: 10px 20px;
+    font-size: 16px;
+    display: block;
+    flex-direction: column;
+    gap: 10px;
+    height: 100%;
+}
```

Comment:
```suggestion
.courses-cards-text-container{
    margin: 0;
    padding: 20px;
    font-size: 16px;
    display: flex;
    flex-direction: column;
    gap: 10px;
    height: 100%;
}
```
![image](https://github.com/JunPlayGround/Frontend-Basic-SIte/assets/83479175/e837f63a-7e0a-471f-97a7-729053f51cd0)
開発者ツールで薄く表示されている宣言は、それが無効であることを示しています。
gapを有効化させるために、flexboxにしましょう。

また、特に理由がなければpaddingはfigmaと同じく四方20pxでいいかなと思います。

Reply:
幅を狭くした時に文字がかぶるのが嫌だったので縦方向のpaddingを狭くしました

Reply:
細かいところまで見れていて素晴らしいです！
実際の案件でもこのようにfigmaページに落とし込む際にデザイナーの想定外の事態が起こることはあるので、今回のようにプルリクエストで理由をコメントしておくのが良いです。
下のようなスクリーンショットを添えるとレビュアーに伝わりやすいのでぜひやってみてください 🙌 

参考：
![image](https://github.com/JunPlayGround/Frontend-Basic-SIte/assets/83479175/356ef468-c567-4919-b1e2-a2693658f8a7)

Reply:
ありがとうございます。